backend/src/main/java/com/bakdata/conquery/mode/local/LocalNamespaceHandler.java

```diff
@@ -42,7 +68,12 @@ public LocalNamespace createNamespace(NamespaceStorage namespaceStorage, MetaSto
 
 	@Override
 	public void removeNamespace(DatasetId id, LocalNamespace namespace) {
-		// nothing to do
+		try {
+			namespace.getDslContextWrapper().close();
```

Comment:
Warum packst du das nicht in die namespace#close()?

Reply:
Ah stimmt, verschiebe ich in den LocalNamespace 👍